mllib/src/main/scala/org/apache/spark/ml/regression/GeneralizedLinearRegression.scala

```diff
@@ -284,6 +288,7 @@ class GeneralizedLinearRegression @Since("2.0.0") (@Since("2.0.0") override val
         .setParent(this))
     val trainingSummary = new GeneralizedLinearRegressionTrainingSummary(dataset, model,
       irlsModel.diagInvAtWA.toArray, irlsModel.numIterations, getSolver)
+    instr.logSuccess(model)
```

Comment:
We need to log success if we enter the `WeightedLeastSquares` branch above as well. Maybe we can refactor the code to use `if else` instead of an `if` with a `return`. That way we only need to log success in one place.